test/Makefile

```diff
@@ -2,7 +2,7 @@ all: run
 
 run:
 	@echo "Running all tests..."
-	@python3 -m pytest -v --durations=20 .
+	@python3 -m pytest -v -s -n auto --durations=20 .
```

Comment:
I don't think we should use `-s`, except for debugging.

Reply:
Fixed in c2cece69f45c92dc0471dbf09c330054c3bfb14a.